packages/editor/src/components/post-title/index.native.js

```diff
@@ -32,10 +32,6 @@ class PostTitle extends Component {
 		this.props.onUnselect();
 	}
 
-	focus() {
```

Comment:
Currently, we pass a ref to this component up to the editor and use this `focus` method in [the Editor](https://github.com/WordPress/gutenberg/blob/update/editor/post-title-in-store/packages/edit-post/src/editor.native.js#L76):

```
this.subscriptionParentSetFocusOnTitle = subscribeSetFocusOnTitle( () => {
	if ( this.postTitleRef ) {
		this.postTitleRef.focus();
	}
} );
```

I'm very excited that your change will allow us to stop passing the ref up there and make that call (the Editor could just directly emit your new `TOGGLE_POST_TITLE_SELECTION` action now without needing the ref). Until we make that change though we need to keep this focus method because the app will try to call that method to focus the title anytime an empty new post is opened.

Reply:
Indeed, good catch 👍